samples/cpp/stereo_calib.cpp

```diff
@@ -34,114 +32,155 @@
 #include <stdlib.h>
 #include <ctype.h>
 
-using namespace cv;
+    using namespace cv;
 using namespace std;
 
 static int print_help(char** argv)
 {
     cout <<
-            " Given a list of chessboard images, the number of corners (nx, ny)\n"
-            " on the chessboards, and a flag: useCalibrated for \n"
-            "   calibrated (0) or\n"
-            "   uncalibrated \n"
-            "     (1: use stereoCalibrate(), 2: compute fundamental\n"
-            "         matrix separately) stereo. \n"
-            " Calibrate the cameras and display the\n"
-            " rectified results along with the computed disparity images.   \n" << endl;
-    cout << "Usage:\n " << argv[0] << " -w=<board_width default=9> -h=<board_height default=6> -s=<square_size default=1.0> <image list XML/YML file default=stereo_calib.xml>\n" << endl;
+        " Given a list of chessboard images, the number of corners (nx, ny)\n"
+        " on the chessboards, and a flag: useCalibrated for \n"
+        "   calibrated (0) or\n"
+        "   uncalibrated \n"
+        "     (1: use stereoCalibrate(), 2: compute fundamental\n"
+        "         matrix separately) stereo. \n"
+        " Calibrate the cameras and display the\n"
+        " rectified results along with the computed disparity images.   \n" << endl;
+    cout << "Usage:\n " << argv[0] << " -w=<board_width default=9> -h=<board_height default=6>"
```

Comment:
Need to add note that the board size is defined in inner corners for chessboard and in units for all other cases.